js/index.js

```diff
@@ -1,23 +1,28 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  const price = product.querySelector(".price span").innerHTML;
+  const quantity = product.querySelector(".quantity input").value;
+  const subtotal = Number(price) * parseFloat(quantity);
+  const subtotalElement = product.querySelector(".subtotal span");
+  subtotalElement.innerHTML = subtotal;
+  return subtotal;
 }
 
+
+// ITERATION 2
+
 function calculateAll() {
-  // code in the following two lines is added just for testing purposes.
-  // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
-  // end of test
+  const cartItemElements = document.getElementsByClassName("product");
 
-  // ITERATION 2
-  //... your code goes here
+  let totalValue = 0;
+
+  for (cartItemElement of cartItemElements) {
+    totalValue += updateSubtotal(cartItemElement);
+  }
 
   // ITERATION 3
-  //... your code goes here
+  document.querySelector("#total-value span").innerHTML = totalValue;
 }
 
```

Comment:
Good stuff Roy-Arne!
I only have one comment regarding the blank line.
The code looks good.